<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>584</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>671</width>
      <height>131</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25 150pt &quot;Roboto&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Up_BT">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Down_BT">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Close_BT">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>370</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Set_BT">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>86</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>start</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>first</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>second</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>third</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>491</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>370</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt; Next</string>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25 150pt &quot;Roboto&quot;;</string>
    </property>
   </widget>
   <zorder>textEdit</zorder>
   <zorder>Close_BT</zorder>
   <zorder>comboBox</zorder>
   <zorder>Up_BT</zorder>
   <zorder>Down_BT</zorder>
   <zorder>Set_BT</zorder>
   <zorder>textEdit_2</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>textEdit_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>dial</zorder>
   <zorder>textEdit_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
